test/support/stack.ex

```diff
@@ -8,7 +8,7 @@ defmodule Stack do
     Agent.update(__MODULE__, &[item | &1])
   end
 
-  def pop(item) do
+  def pop do
```

Comment:
How it is possible that the code is working with this change? Is this module used at all?

Reply:
I had the same question. ;)  The short answer is that the `item` parameter is never used — the `item` on the next line is a completely different variable that's only scoped for the lambda that's passed to `get_and_update`.

I don't see any calls to `pop`, though, and `Stack` is only used by the `Pinger` middleware module, which is itself unused (afaict). There's a bit of dead code, I believe.